### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230831

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#37] [Announcements]
This is an offline standalone installer for a modified version of AUTOMATIC1111's Stable Diffusion web UI that includes experimental changes and uses the nightly PyTorch build from 8/30. Note that this is an test build so crashes and bugs should be expected. All dependencies needed to start using Stable Diffusion (other than the model) are included and this does not require Xcode command line tools to be installed.

This release currently only has the installer for Apple silicon Macs, an installer for Intel Macs will be included with 1.6.0. macOS 13.5 or later is highly recommended, older versions of macOS are untested.

Also included now is [a fork of the ControlNet extension](https://github.com/brkirch/sd-webui-controlnet/tree/094b4d20ddfce7c2ea3b50c290313620e25a8c54) which is currently Mikubill/sd-webui-controlnet#1952 with Mikubill/sd-webui-controlnet#1950 merged into it. Note that since this is from a draft PR that is a major rework of the ControlNet extension there could still be some bugs.

The `--electron` command line option is still available, but it is no longer the default. To make it the default, open `webui-user.sh` in Xcode or any text editor and change
```shell
#export COMMANDLINE_ARGS=""
```
to
```shell
export COMMANDLINE_ARGS="$COMMANDLINE_ARGS --electron"
```

The 1.6.0 installer will be available soon. It is currently delayed until SDXL support is merged into the main branch of the ControlNet extension so that it can be included. That will likely happen within the next week.

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230831_experimental'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230831</a>.</em>

**Comment by Max1mu1** [comment]
Thanks for it!

**Comment by escoolioinglesias** [comment]
You are awesome! Super excited for 1.6.0 and the better SDXL ControlNet support <3
Thank you for making it a breeze to install 🙌

**Comment by fckbauer** [comment]
awesome news bro. your efforts are much appreciated 🙏🏻
